Jenkinsfile_CNP

```diff
@@ -1,13 +1,117 @@
 #!groovy
-
+import uk.gov.hmcts.contino.AppPipelineDsl
+properties(
+        [
+                [
+                        $class       : 'GithubProjectProperty',
+                        projectUrlStr: 'https://github.com/hmcts/civil-sdt-commissioning'
+                ],
+                pipelineTriggers([[$class: 'GitHubPushTrigger']])
+        ]
+)
 @Library("Infrastructure")
 
+import uk.gov.hmcts.contino.GradleBuilder
+
 def type = "java"
 def product = "civil"
 def component = "sdt-commissioning"
 
+GradleBuilder builder = new GradleBuilder(this, product)
+
+def secrets = [
+        's2s-${env}': [
+                secret('microservicekey-civil-sdt-commissioning', 'SDT_COMMISSIONING_S2S_SECRET')
+        ],
```

Comment:
we have not yet integrated with s2s, we don't need this at the moment... and commissioning doesn't need this at all as we are not going to integrate with the any other services from commissioning module (not going to interact with civil)

Reply:
Good